packages/build-utils/src/convert-runtime-to-plugin.ts

```diff
@@ -29,6 +29,24 @@ const shouldIgnorePath = (
   return isNative || ignoreFilter(file);
 };
 
+const getSourceFiles = async (workPath: string, ignoreFilter: any) => {
+  const list = await glob('**', {
+    cwd: workPath,
+  });
+
+  // We're not passing this as an `ignore` filter to the `glob` function above,
+  // so that we can re-use exactly the same `getIgnoreFilter` method that the
+  // Build Step uses (literally the same code). Note that this exclusion only applies
+  // when deploying. Locally, another exclusion further below is needed.
```

Comment:
What does "futher below" mean? Seems like there is nothing below this loop

Reply:
Great point. I moved that comment, but didn't update it.

Updated it in https://github.com/vercel/vercel/pull/7117/commits/7fbf4668cc74168d33be2fe74b76ad1def2ed91b!